metadata.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": [
     {
       "name": "puppetlabs-stdlib",
-      "version_requirement": ">= 4.13.1 < 6.0.0"
+      "version_requirement": ">= 4.17.0 < 6.0.0"
```

Comment:
can you pull this into a separate PR? It isn't 100% related to the actual issue here in this PR and it would give us a dedicated entry in the CHANGELOG.md. Same for the `puppet-nodejs` dependency